composer.json

```
@@ -6,7 +6,7 @@
     "require": {
         "php": ">=7.4",
         "ext-json": "*",
-        "bunny/bunny": "^0.4.3",
+        "bunny/bunny": "^0.5.1",
```

Given that 0.5.0 and 0.5.1 are API compatible with 0.4.3 from what I can see by looking at the changes, and that our code should work with both 0.4.3 and 0.5.0 and up (0.5.0 contains the PHP 8 requirements so it also works). I think we should just add the new version to allow updates, while also making sure we don't unintentionally break consumers who may be stuck on 0.4.3 for other reasons.

```suggestion
        "bunny/bunny": "^0.4.3 || ^0.5.0",
```